test/assembler/encoder.test.ts

```diff
@@ -58,6 +58,36 @@ describe('encode', function () {
     expect(file.content[0].value).toBe(0x77)
     expect(file.content[1].value).toBe(0x44)
   })
+  it('should encode DCB instruction with Strings', function () {
+    const code: ICodeFile = {
+      symbols: {},
+      areas: [
+        {
+          type: AreaType.Data,
+          isReadOnly: true,
+          name: '|.data|',
+          instructions: [
+            {
+              name: 'DCB',
+              options: ['"TestString"', `"TestString with Escape'""`],
```

Comment:
perhaps add additional escape in middle of string

Reply:
Done